com.unity.render-pipelines.high-definition/Documentation~/UI-Best-Practices.md

```diff
@@ -1,14 +1,20 @@
 # Working with UI in the High Definition Render Pipeline
 
-Almost every Project uses some form of user interface (UI). This could be: in a menu system, to indicate a point of interest in your application, or to display valuable information to users. This guide explains how to use [Unity UI](https://docs.unity3d.com/Packages/com.unity.ugui@latest) in your High Definition Render Pipeline (HDRP) Project as well as provide best practice information to keep in mind when working with it.
+Almost every Project uses some form of user interface (UI). This could be: 
+
+* In a menu system
+* To specify a point of interest in your application
+* To display valuable information to users. 
+
+This guide explains how to use [Unity UI](https://docs.unity3d.com/Packages/com.unity.ugui@latest) in your High Definition Render Pipeline (HDRP) Project and provide best practice information to keep in mind when working with it.
```

Comment:
The end of the last setence is... "to keep in mind when working with it" wordy and useless imo. Just remove it :)